src/Illuminate/Console/Scheduling/CallbackEvent.php

```diff
@@ -58,6 +58,8 @@ public function run(Container $container)
 
         try {
             $response = $container->call($this->callback, $this->parameters);
+        } catch (Exception $e) {
```

Comment:
Looks strange to provide a throw docblock and hiding the throwing of this exception